rector.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+use Rector\Config\RectorConfig;
+use Rector\Set\ValueObject\LevelSetList;
+
+return static function (RectorConfig $rectorConfig): void {
+    $rectorConfig->paths([
+        __DIR__ . '/src',
+        __DIR__ . '/tests',
+    ]);
+
+    // define sets of rules
```

Comment:
Indentation?

Reply:
Usher follows [PSR-12](https://www.php-fig.org/psr/psr-12/). 4 spaces.

Reply:
Right, but this should still be aligned with line of code beneath it, right?

Reply:
Ahh, it's actually the code below that is indented incorrectly. I will take care of it. Thanks for catching!